### DISCUSSION TITLE
Difference between self.position.close() and self.sell()

### DISCUSSION THREAD

**Discussion by alexmulo** [kernc/backtesting.py#1156] [General]
Dear all,

I had a look in the documentation and in the code but I could find the difference between the above methods. Could you please explain me when I use the one or the other? Do you have any example? Thanks

**Comment by ghost** [comment]
self.sell() =  take short position
self.position.close() = exit position

**Reply by alexmulo** [reply]
what do you mean "take short position"? Do you mean short selling or just step partially back from the market?

**Reply by mtrfb6** [reply]
A short position would be a short sale, yes.
